### PR TITLE
Landing page, and updated SCSS Vars

### PR DESCRIPTION
**Pull request by ostaron**
This PR also changes the name of $grey-labels to $grey-text, which is more accurate; and updates the Checkbox component to use the new scss vars. 

The Demo section still needs to be built.

### CODE REVIEW

File: src/Main/MainFooter.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+              <Tag color="pink" content={<a href="#">@GarretLevine</a>} />
+              <Tag color="pink" content={<a href="#">@KevanH</a>} />
+              <Tag color="pink" content={<a href="#">@SeafoamGreenway</a>} />
+            </div>
+          </div>
+        </div>
+      </footer>
+    )
+  }
+}
```
Comment (GarrettLevine) [inline src/Main/MainFooter.jsx]: new line at end of file please :see_no_evil:

File: src/Main/MainFooter.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    padding: 50px;
+}
+
+.footer_heading {
+    color: white;
+    font-weight: 700;
+    margin-bottom: 12px;
+    font-size: $font-size-landing_regular;
+    line-height: $line-height-landing_regular
+}
```
Comment (GarrettLevine) [inline src/Main/MainFooter.scss]: new line

File: src/Main/MainFooter.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+@import '../styles/main.scss';
+
+.footer {
+    background-color: #F17CC6;
```
Comment (GarrettLevine) [inline src/Main/MainFooter.scss]: let's put this colour in a variable

File: src/Main/MainFooter.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
+@import '../styles/main.scss';
+
+.footer {
+    background-color: #F17CC6;
+}
+
+.footer_background-container {
+    background: url('~/assets/imgs/icons.png') repeat;
```
Comment (GarrettLevine) [inline src/Main/MainFooter.scss]: nice :ok_hand: I wouldn't have really known what to do for this background.

File: src/Main/MainFooter.scss [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+.footer_background-container {
+    background: url('~/assets/imgs/icons.png') repeat;
+    height: 100%;
+    width: 100%;
+    padding: 50px;
+}
+
+.footer_heading {
+    color: white;
+    font-weight: 700;
```
Comment (GarrettLevine) [inline src/Main/MainFooter.scss]: should we also store basic font weights in SCSS variables?

File: src/Main/MainHeader.scss [hunk 6]
```diff
@@ -0,0 +1,33 @@
+@import '../styles/main.scss';
+
+.mainheader {
+    max-width: 860px;
```
Comment (GarrettLevine) [inline src/Main/MainHeader.scss]: I haven't looked though a lot of the file, but a bunch of the max width the same across multiple 'inner wrappers', if so let's variable it upppp - if not IGNORE ME.

File: src/Main/MainHeader.scss [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    display: flex;
+    justify-content: space-between;
+    @media screen and (max-width: $break-medium) {
+        margin: 0 30px;
+    }
+}
+
+
+.mainheader_logo {
+    flex: 0 1 29%;
```
Comment (GarrettLevine) [inline src/Main/MainHeader.scss]: this is FINE but I am just curious to know how you arrived at this 29% number?

File: src/Main/NavItem.scss [hunk 8]
```diff
@@ -0,0 +1,21 @@
+@import '../styles/main.scss';
+
+.navitem {
+    margin-left: 60px;
+    list-style: none;
+    @media screen and (max-width: 700px) {
```
Comment (GarrettLevine) [inline src/Main/NavItem.scss]: personal stylistic preference, but I like my SCSS organized like so 

1. base styles
2. pseudo styles
3. Media queries

Let me know your thoughts.

File: src/Main/main.scss [hunk 9]
```diff
@@ -1,5 +1,17 @@
...
+    padding: 0;
+    margin: 0;
+}
 
-h1 {
-    color: $pink-medium;
-}
\ No newline at end of file
+body {
+    margin: 0;
```
Comment (GarrettLevine) [inline src/Main/main.scss]: should we maybe throw this in the `index.scss` or normalize file? Is this something that should be on all pages?

File: src/Main/MainHeader.scss (hunk 6 above)
Comment (ostaron) [reply src/Main/MainHeader.scss]: Only the header is this width - the rest of the sections are all 800px.

File: src/Main/MainHeader.scss (hunk 7 above)
Comment (ostaron) [reply src/Main/MainHeader.scss]: Hahaha, because that's precisely the %width of the logo in mark's design.

File: src/Main/NavItem.scss (hunk 8 above)
Comment (ostaron) [reply src/Main/NavItem.scss]: That's logical, considering psuedo elements could potentially be altered in a media query.

File: src/Main/main.scss (hunk 9 above)
Comment (ostaron) [reply src/Main/main.scss]: Ah - I put this on before we included the normalize file. I see that the same rule is included in that. I'll remove this.

File: src/Main/MainFooter.scss (hunk 5 above)
Comment (ostaron) [reply src/Main/MainFooter.scss]: Alternative to that: I just tried font-weight: bold; and it results in the same thing, and is more semantic.

**Pull request merged**